python-flask/app/main/routes.py

```diff
@@ -103,10 +103,11 @@ def unfollow_user(username):
     existing_follower = Follower.query.filter_by(who_id=g.user.user_id,
                                                  whom_id=whom.user_id).first()
     if existing_follower is None:
-        flash(f'You are no longer following "{username}"')
+        flash(f'You are not following "{username}"')
```

Comment:
I think what you had before is what helge has in his service

```
 flash('You are no longer following "%s"' % username)
```

Reply:
https://github.com/search?q=repo%3AHelgeCPH%2Fcpython_energy+You+are+not+following&type=code

Reply:
Yea I wasn't sure. I will just throw a 404 error for now. Don't know what you guys doing in that case